Reflections/Week_5_Reflection.md

```diff
@@ -0,0 +1,22 @@
+# Weekly Reflection- Week Five
+
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  * I have finished the interiors for the mod. I have made the interiors accessible in game. I have finished navmeshing (movement pathing for AI) for the interiors. I have finished the companion perks that are rewarded on completion of the quest. I have made the quest finishable. It is important to note that certain things are going to have to get cut for the sake of time. I have decided that voiced NPCs are too time intensive to be done. Also, I have scrapped the dialogue between Mario and his boss for the same reason.
+
+![Main Office](WeeklyScreenshots/WeekFiveMainOffice.png)
+![Mario's Office](WeeklyScreenshots/WeekFiveMariosOffice.png)
+![Mario Fired Perk](WeeklyScreenshots/WeekFiveMarioFiredPerk.png)
+![Mario Promotion Perk](WeeklyScreenshots/WeekFiveMarioPromotionPerk.png)
+![Return Notes One](WeeklyScreenshots/WeekFiveReturnNotesOne.png)
+![Return Notes Two](WeeklyScreenshots/WeekFiveReturnNotesTwo.png)
+![Return Notes Three](WeeklyScreenshots/WeekFiveReturnNotesThree.png)
```

Comment:
Great show of progress here as well! As always, thank you for sharing these alongside your weekly reflection. It lends a great perspective on progress I wouldn't otherwise have.